pkg/tanka/workflow.go

```diff
@@ -17,18 +17,18 @@ import (
 func Apply(baseDir string, mods ...Modifier) error {
 	opts := parseModifiers(mods)
 
-	p, err := parse(baseDir, opts)
+	l, err := load(baseDir, opts)
```

Comment:
nit: I see the `load`-ed value represented by `p`, `r` and `l` -- is there any special meaning to those letters?